SPECS-EXTENDED/reaper/reaper.spec

```diff
@@ -0,0 +1,156 @@
+Summary: Cassandra Reaper
```

Comment:
Standard comment for new specs, please ignore if already done: please run a spec linter through this file. The linter is not perfect, so please also move the `%global` and `%define` declarations to the top of the file. Thanks!

Reply:
Addressed.